JavaDirectory/src/test/resources/application.yml

```diff
@@ -0,0 +1,15 @@
+spring:
+  datasource:
+    drive-class-name: org.h2.Driver
+#          url: jdbc:h2:mem:teamC_josh;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;MODE=MYSQL;DATABASE_TO_UPPER=false;IGNORECASE=TRUE
```

Comment:
This comment seems to be identical to the line below

Reply:
done